LICENSE.rst

```diff
@@ -1,17 +1,24 @@
 BSD 3-Clause License
 
-Copyright 2014-2015, Board of Trustees of Michigan State University
-          2016-2024, The Trustees of Columbia University in the City of New York.
+Copyright (c) 2006-2007, Board of Trustees of Michigan State University
```

Comment:
I think this may also be correct as it was before because Luke was still at MSU when he was working on this. Please change back